linode_api4/objects/database.py

```diff
@@ -128,6 +137,143 @@ class PostgreSQLDatabaseBackup(DatabaseBackup):
     api_endpoint = "/databases/postgresql/instances/{database_id}/backups/{id}"
 
 
+@dataclass
+class MySQLDatabaseConfigMySQLOptions(JSONObject):
+    """
+    MySQLDatabaseConfigMySQLOptions represents the fields in the mysql
+    field of the MySQLDatabaseConfigOptions class
+    """
+
+    connect_timeout: Optional[int] = None
+    default_time_zone: Optional[str] = None
+    group_concat_max_len: Optional[float] = None
+    information_schema_stats_expiry: Optional[int] = None
+    innodb_change_buffer_max_size: Optional[int] = None
+    innodb_flush_neighbors: Optional[int] = None
+    innodb_ft_min_token_size: Optional[int] = None
+    innodb_ft_server_stopword_table: Optional[str] = None
+    innodb_lock_wait_timeout: Optional[int] = None
+    innodb_log_buffer_size: Optional[int] = None
+    innodb_online_alter_log_max_size: Optional[int] = None
+    innodb_print_all_deadlocks: Optional[bool] = None
+    innodb_read_io_threads: Optional[int] = None
+    innodb_rollback_on_timeout: Optional[bool] = None
+    innodb_thread_concurrency: Optional[int] = None
+    innodb_write_io_threads: Optional[int] = None
+    interactive_timeout: Optional[int] = None
+    internal_tmp_mem_storage_engine: Optional[str] = None
+    log_output: Optional[str] = None
+    long_query_time: Optional[float] = None
+    max_allowed_packet: Optional[int] = None
+    max_heap_table_size: Optional[int] = None
+    net_buffer_length: Optional[int] = None
+    net_read_timeout: Optional[int] = None
+    net_write_timeout: Optional[int] = None
+    slow_query_log: Optional[bool] = None
+    sort_buffer_size: Optional[int] = None
+    sql_mode: Optional[str] = None
+    sql_require_primary_key: Optional[bool] = None
+    tmp_table_size: Optional[int] = None
+    wait_timeout: Optional[int] = None
```

Comment:
Do these fields correspond to the fields returned by `DatabaseGroup(...).mysql_config_options()`?

I think having explicit fields is still the right call if so, but we also need to make sure (unit test?) users can access any fields added to that endpoint in the future.